linkis-engineconn-plugins/spark/pom.xml

```diff
@@ -418,6 +454,18 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>${hadoop-hdfs-client-shade.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>${hadoop-hdfs-client-shade.artifact}</artifactId>
+      <version>${hadoop-hdfs-client-shade.version}</version>
```

Comment:
can there remove the version property? or use ${hadoop.version}

Reply:
compile will fail if no version specific, ${hadoop.version} seems more reasonable.